fab/environments.yml

```diff
@@ -87,90 +77,75 @@ production:
   environment: production
   should_migrate: True
 
-  sms_queue_enabled: True
-  pillow_retry_queue_enabled: True
-  celery_periodic_enabled: True
-  reminder_queue_enabled: True
-  reminder_rule_queue_enabled: True
-  reminder_case_update_queue_enabled: True
-
   new_relic_enabled:
     - hqdjango3.internal.commcarehq.org
 
-  celery_params:
-    main:
-      concurrency: 8
-    background_queue:
-      concurrency: 4
-    periodic:
-      concurrency: 4
-    pillow_retry_queue:
-      concurrency: 1
-    reminder_case_update_queue:
-      concurrency: 4
-    reminder_queue:
-      concurrency: 8
-    reminder_rule_queue:
-      concurrency: 4
-    saved_exports_queue:
-      concurrency: 3
-    sms_queue:
-      concurrency: 8
+  celery_processes:
+    hqcelery0:
+      main:
+        concurrency: 8
+      background_queue:
+        concurrency: 4
+      periodic:
+        concurrency: 4
+      pillow_retry_queue:
+        concurrency: 1
+      reminder_case_update_queue:
+        concurrency: 4
+    hqcelery1:
+      reminder_queue:
+        concurrency: 8
+      reminder_rule_queue:
+        concurrency: 4
+      saved_exports_queue:
+        concurrency: 3
+      sms_queue:
+        concurrency: 8
+      flower: {}
 
 staging:
   environment: staging
   should_migrate: True
 
-  pillow_retry_queue_enabled: True
-
   new_relic_enabled:
     - hqdjango0-staging.internal.commcarehq.org
     - hqdjango1-staging.internal.commcarehq.org
     - hqdb0-staging.internal.commcarehq.org
 
-  celery_params:
-    main:
-      concurrency: 4
-    background_queue:
-      concurrency: 2
-    pillow_retry_queue:
-      concurrency: 1
-    reminder_case_update_queue:
-      concurrency: 1
-    reminder_queue:
-      concurrency: 1
-    reminder_rule_queue:
-      concurrency: 1
-    saved_exports_queue:
-      concurrency: 3
+  celery_processes:
+    '*':
+      main:
+        concurrency: 4
+      background_queue:
+        concurrency: 2
+      pillow_retry_queue:
+        concurrency: 1
+      saved_exports_queue:
+        concurrency: 3
+      flower: {}
```

Comment:
Does just not having queues show up in this list replace e.g. `sms_queue_enabled`?